packages/plugin-core/src/components/views/NoteGraphViewFactory.ts

```diff
@@ -28,6 +31,7 @@ export class NoteGraphPanelFactory {
     ext: DendronExtension,
     engineEvents: EngineEventEmitter
   ): vscode.WebviewPanel {
+    const start = process.hrtime();
```

Comment:
This should be within the `if (!this._panel)` bracket right? Otherwise it will reset if someone uses the show note graph command again while they already have the note graph open?

Reply:
Used [OpenWebviewAndMeasure](https://github.com/dendronhq/dendron/blob/e3e7f15a503d222c68bef9e0f6b6c8973b0f58b2/packages/plugin-core/src/views/utils.ts#L288) utility to measure timeOpen.